builder/builder_test.go

```diff
@@ -2,16 +2,48 @@ package builder_test
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 
 	"github.com/ulule/loukoum/builder"
 	"github.com/ulule/loukoum/stmt"
 )
 
-func Failure(is *require.Assertions, callback func() builder.Builder) {
-	is.Panics(func() {
-		_ = callback().String()
+var now = time.Now()
```

Comment:
Why do you need `time.Now()` here ?

Reply:
Ok I see why you need that. I've a suggestion for you so we don't need that global variable using `BuilderTestCase` _(see below)_.

Reply:
Global variables are usually acceptable in tests.

Reply:
Ok why not, not a big fan but let's do that.